examples/gnrc_networking/README.md

```diff
@@ -80,16 +86,17 @@ Now, on the RIOT side, send a UDP packet using:
 
     udp send fe80::4049:5fff:fe17:b3ae 8808 testmessage
 
-You should see `testmessage` appear in netcat. Instead of using netcat, you can of
-course write your own software, but you may have to bind the socket to a specific
-interface (tap0 in this case). For an example that shows how to do so, see
-[here](https://gist.github.com/backenklee/dad5e80b764b3b3d0d3e).
+You should see `testmessage` appear in netcat. Instead of using netcat,
+you can of course write your own software, but you may have to bind the
+socket to a specific interface (tap0 in this case). For an example that
+shows how to do so, see [here](https://gist.github.com/backenklee/dad5e80b764b3b3d0d3e).
```

Comment:
Maybe make this a reflink, so the line is ended properly.

```suggestion
shows how to do so, see [here][tap-socket]

[tap-socket]: (https://gist.github.com/backenklee/dad5e80b764b3b3d0d3e).
```

BTW the link leads to a 404 :(.

Reply:
Aw shoot! Good catch. @backenklee , do you happen to have a copy of this gist's contents? ;)

Reply:
nope, sorry :( but all that code did was basically this: https://stackoverflow.com/questions/14478167/bind-socket-to-network-interface#14478657

Reply:
haha ok, will do :D